selfdrive/controls/lib/events.py

```diff
@@ -579,7 +586,7 @@ def joystick_alert(CP: car.CarParams, sm: messaging.SubMaster, metric: bool) ->
   },
 
   EventName.wrongGear: {
-    ET.SOFT_DISABLE: SoftDisableAlert("Gear not D"),
+    ET.SOFT_DISABLE: UserSoftDisableAlert("Gear not D"),
     ET.NO_ENTRY: NoEntryAlert("Gear not D"),
```

Comment:
I like how warnings give actions, like "touch wheel" and "take control".

Maybe NO_ENTRY should too, because action is required to engage: "Shift to Drive", "Close doors", "Fasten seatbelts"

Reply:
For soft disable, it's talking about past-tense "what happened", so those sound right.

Also I don't know the nitty gritty here so call out if I'm missing deets.

Reply:
I think it's fine as is since it's more a response to the engagement attempt. @alexcomma what do you think?